metricbeat/docs/modules/gcp.asciidoc

```diff
@@ -346,6 +346,7 @@ metricbeat.modules:
   credentials_file_path: "your JSON credentials file path"
   exclude_labels: false
   period: 1m
+  location_label: "resource.labels.zone"
```

Comment:
This is not a required field is not it? If not maybe to put it in comment?

Reply:
Yeah, this is an optional field. I'll include that in documentation as well